### DISCUSSION TITLE
What is 'refs/remotes/origin/palette' ?

### DISCUSSION THREAD

**Discussion by FredericVansteenkiste** [community/community#22291] [New to GitHub]
<p>Hello everyone,</p>
<p>I am using git and I recently create a branche “palette” on my project. Satisfy by my new developpement, I merged everything in my branche “master”</p>
<pre><code>$ git branch
  master
* palette
$ git checkout master
Basculement sur la branche 'master'
Votre branche et 'origin/master' ont divergé,
et ont 6 et 2 commits différents chacune respectivement.
  (utilisez "git pull" pour fusionner la branche distante dans la vôtre)
$ git merge palette
</code></pre>
<p>Everything went OK, the result in my branche “master” is correct and correspond to what I was expected.</p>
<p>Then I decided to close my branch “palette” which was no more usefull, but here is the result I get :</p>
<pre><code>$ git branch -d palette
warning: branche 'palette' non supprimée car elle n'a pas été fusionnée dans
         'refs/remotes/origin/palette', même si elle est fusionnée dans HEAD.
error: La branche 'palette' n'est pas totalement fusionnée.
Si vous êtes sur que vous voulez la supprimer, lancez 'git branch -D palette'.
</code></pre>
<p>which I could translate by : the branch “palette” was not closed because it is not merged in ‘refs/remotes/origin/palette’, even if it is merged in HEAD. error: the branche “palette” is not completly merged.</p>
<p>What am I supposed to do to completly merge my branch ? what is ‘refs/remotes/origin/palette’ ?</p>
<p>Thanks for your help …</p>

**Comment by FredericVansteenkiste** [comment, marked as answer]
<p>I suppress and commit everything and then I get back all the files and that way I finally succed.</p>
<p>Probably somehing better to do but I didn’t found it …</p>
<p>How do we close a subject ?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/fredericvansteenkist">@fredericvansteenkist</a>,</p>
<p>Thanks for being here and we appreciate you coming back to post what worked for you. We’d like to keep this open as it might help another user if that’s ok?</p>
<p>Best,</p>
<p>Andrea</p>
